corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -77,6 +77,32 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
     if (el.length) {
         el.koApplyBindings(releasesMain);
         _.defer(function () { releasesMain.goToPage(1); });
+
+        var releaseControlEl = $('#release-control');
+        if (releaseControlEl.length) {
+            releasesMain.toggleReleasesControl(false);
```

Comment:
nit: Feel free to ignore this if don't feel worth it. I think I shared this feedback earlier as well.
This would be better named as "showReleaseControl".

Reply:
Ok. I think you are referring to this comment from earlier? https://github.com/dimagi/commcare-hq/pull/32245#discussion_r1013287887

Reply:
Ya, `showReleaseControls` is even better than `setReleaseLockButtons`